docker-local/postman/scripts/_00_seed_all.sh

```diff
@@ -14,6 +14,7 @@ set -u
 ./_03_seed_dfspa.sh
```

Comment:
FYI - I'm currently working on replacing this tower of cards with `ml-bootstrap`

Reply:
🙏